mavros_msgs/msg/OrbitStatus.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp # time since system start (microseconds)
+float32 radius   # Radius of the orbit circle. Positive values orbit clockwise, negative values orbit counter-clockwise. [m]
+uint8 frame      # The coordinate system of the fields: x, y, z.
+float64 x        # X coordinate of center point. Coordinate system depends on frame field: local = x position in meters * 1e4, global = latitude in degrees * 1e7.
+float64 y        # Y coordinate of center point. Coordinate system depends on frame field: local = y position in meters * 1e4, global = latitude in degrees * 1e7.
+float32 z        # Altitude of center point. Coordinate system depends on frame field.
```

Comment:
Perhaps better to use GeoPoint or Point?

Reply:
I don't have a strong opinion on that, but I try to keep API as much as similar wrt. https://github.com/mavlink/mavlink/blob/2571f450f576d7b0463c0c8eb57eeb6e6f0a5bd8/message_definitions/v1.0/common.xml#L6575-L6577

Reply:
@tahsinkose yes but this is a bridge to ROS - So ROS data types should be used whenever possible.